centreon/packages/ui/src/Dialog/Duplicate/index.tsx

```diff
@@ -1,22 +1,32 @@
 import { useState } from 'react';
 
-import { isEmpty } from 'ramda';
+import { isEmpty, or } from 'ramda';
+import { makeStyles } from 'tss-react/mui';
 
 import TextField from '@mui/material/TextField';
 
 import Dialog, { Props as DialogProps } from '..';
 
 type Props = DialogProps & {
   labelInput?: string;
+  limit?: number;
 };
 
+const useStyles = makeStyles()((theme) => ({
+  container: {
+    width: theme.spacing(30)
+  }
+}));
+
 const Duplicate = ({
   labelInput = 'Count',
   labelConfirm = 'Duplicate',
   labelTitle = 'Duplicate',
+  limit = Infinity,
```

Comment:
Does the undefined value work?

Reply:
You should take into account that the limit might ne undefined then

Reply:
if the limit props were not given then the value is the default one: Infinity

Reply:
and I have verified this in the other stories

Reply:
Or you can a little bit of code to handle undefined

Reply:
I don't understand! I already did this. Is it a bad practice to use a default value ?

Reply:
Both are valid. We can keep this way